content/en/docs/tasks/traffic-management/locality-load-balancing/before-you-begin/index.md

```diff
@@ -26,6 +26,36 @@ We will deploy several instances of the `HelloWorld` application as follows:
     caption="Setup for locality load balancing tasks"
     >}}
 
+{{< tip >}}
+In a single multi-zone cluster environment, locality load balancing can also be configured for failover to a different zone within the same cluster.
+To test it, you will need to create a cluster with multiple worker zones and deploy an istiod instance and the app to each zone.
+
+1) If you don’t have a multi-zone Kubernetes cluster, you can deploy one locally using `kind` with the following command:
+
+  {{< text syntax=bash snip_id=none >}}
+    $ kind create cluster --config=- <<EOF
+    kind: Cluster
+    apiVersion: kind.x-k8s.io/v1alpha4
+    nodes:
+    - role: control-plane
+    - role: worker
+    - role: worker
+    - role: worker
+    EOF
+  {{< /text >}}
+
+2) Use `topology.kubernetes.io/zone` to label each worker with a zone name:
+
+  {{< text syntax=bash snip_id=none >}}
+    $ kubectl label node kind-worker topology.kubernetes.io/zone=us-south10
+    $ kubectl label node kind-worker2 topology.kubernetes.io/zone=us-south12
+    $ kubectl label node kind-worker3 topology.kubernetes.io/zone=us-south13
+  {{< /text >}}
+
+3) Deploy istiod to the control-plane node and the helloworld app to each of the worker nodes.
```

Comment:
Looks like there is a limitation or bug in the page generator for list entries with indented commands inside a tip 😞

Here's my best suggestion given the limitation:

```suggestion
1: If you don’t have a multi-zone Kubernetes cluster, you can deploy one locally using `kind` with the following command:

{{< text syntax=bash snip_id=none >}}
$ kind create cluster --config=- <<EOF
kind: Cluster
apiVersion: kind.x-k8s.io/v1alpha4
nodes:
- role: control-plane
- role: worker
- role: worker
- role: worker
EOF
{{< /text >}}

2: Use `topology.kubernetes.io/zone` to label each worker with a zone name:

{{< text syntax=bash snip_id=none >}}
$ kubectl label node kind-worker topology.kubernetes.io/zone=us-south10
$ kubectl label node kind-worker2 topology.kubernetes.io/zone=us-south12
$ kubectl label node kind-worker3 topology.kubernetes.io/zone=us-south13
{{< /text >}}

3: Deploy istiod to the control-plane node and the helloworld app to each of the worker nodes.
```

Reply:
Thank you! Below is what it looks like after this commit.
![Screenshot 2023-09-05 at 12 09 58 PM](https://github.com/istio/istio.io/assets/36289669/be6f9ada-a539-43db-af65-f020744d5842)